tests/test_utils.py

```diff
@@ -123,42 +136,51 @@ def __marshallable__(self):
     obj = ObjContainer()
     assert utils.to_marshallable_type(obj) == {"foo": 1}
 
+
 def test_is_collection():
     assert utils.is_collection([1, 'foo', {}]) is True
     assert utils.is_collection(('foo', 2.3)) is True
     assert utils.is_collection({'foo': 'bar'}) is False
 
+
 def test_rfcformat_gmt_naive():
     d = dt.datetime(2013, 11, 10, 1, 23, 45)
     assert utils.rfcformat(d) == "Sun, 10 Nov 2013 01:23:45 -0000"
 
+
 def test_rfcformat_central():
     d = central.localize(dt.datetime(2013, 11, 10, 1, 23, 45), is_dst=False)
     assert utils.rfcformat(d) == 'Sun, 10 Nov 2013 07:23:45 -0000'
 
+
 def test_rfcformat_central_localized():
     d = central.localize(dt.datetime(2013, 11, 10, 8, 23, 45), is_dst=False)
     assert utils.rfcformat(d, localtime=True) == "Sun, 10 Nov 2013 08:23:45 -0600"
 
+
 def test_isoformat():
     d = dt.datetime(2013, 11, 10, 1, 23, 45)
     assert utils.isoformat(d) == '2013-11-10T01:23:45+00:00'
 
+
 def test_isoformat_tzaware():
     d = central.localize(dt.datetime(2013, 11, 10, 1, 23, 45), is_dst=False)
     assert utils.isoformat(d) == "2013-11-10T07:23:45+00:00"
 
+
 def test_isoformat_localtime():
     d = central.localize(dt.datetime(2013, 11, 10, 1, 23, 45), is_dst=False)
     assert utils.isoformat(d, localtime=True) == "2013-11-10T01:23:45-06:00"
 
+
 def test_from_datestring():
     d = dt.datetime.now()
     rfc = utils.rfcformat(d)
     iso = d.isoformat()
     assert_date_equal(utils.from_datestring(rfc), d)
     assert_date_equal(utils.from_datestring(iso), d)
 
+
```

Comment:
FWIW, we explicitly ignore the "2-line rule" in setup.cfg because it encourages unnecessary whitespace. This is fine, though.